--- ati-installer_org.sh	2017-05-21 09:47:17.364123956 +0800
+++ ati-installer.sh	2017-05-21 09:58:30.385290059 +0800
@@ -952,8 +952,6 @@
 
 # Script execution starts here
 
-alias echo=/bin/echo
-
 # ensure the working directory is where the script resides
 scriptdir=`dirname "$0"`
 curdir=`pwd`
@@ -1531,31 +1529,6 @@
                     cp -R arch/${ArchDir}/etc/OpenCL/vendors/* \
                           ${TmpDrvFilesDir}/${OPENCL_CONFIG}
                       
-                    ### Copy 32-bit libraries even if using a 64-bit system 
-                    
-                    cp -R arch/x86/usr/X11R6/lib/libfglrx*.* \
-                          ${TmpDrvFilesDir}/${ATI_XLIB_32}
-                    cp -R arch/x86/usr/X11R6/lib/fglrx/fglrx-libGL*.* \
-                          ${TmpDrvFilesDir}/${ATI_XLIB_32}/fglrx
-                    cp -R arch/x86/usr/X11R6/lib/libatiadl*.* \
-                          ${TmpDrvFilesDir}/${ATI_XLIB_32}
-                    cp -R arch/x86/usr/lib/libatiuki*.* \
-                          ${TmpDrvFilesDir}/${ATI_XLIB_32}
-                    cp -R arch/x86/usr/lib/libatical*.* \
-                          ${TmpDrvFilesDir}/${ATI_XLIB_32}                              
-                    cp -R arch/x86/usr/X11R6/lib/libAMDXvBA*.* \
-                          ${TmpDrvFilesDir}/${ATI_XLIB_32}
-                    cp -R arch/x86/usr/X11R6/lib/libXvBAW*.* \
-                          ${TmpDrvFilesDir}/${ATI_XLIB_32}
-                    cp -R arch/x86/usr/X11R6/lib/modules/dri/*.* \
-                          ${TmpDrvFilesDir}/${ATI_3D_DRV_32}
-                    cp -R arch/x86/usr/lib/libamdocl*.* \
-                          ${TmpDrvFilesDir}/${OPENCL_LIB_32}
-                    cp -R arch/x86/usr/lib/libOpenCL*.* \
-                          ${TmpDrvFilesDir}/${OPENCL_LIB_32}
-                    cp -R arch/x86/etc/OpenCL/vendors/* \
-                          ${TmpDrvFilesDir}/${OPENCL_CONFIG}
-                          
                 else
 
                     cp -R arch/${ArchDir}/usr/X11R6/lib/libfglrx*.* \
@@ -1694,6 +1667,9 @@
                     fi
                 fi
 
+		#end of install
+		exit 0
+
                 # replace strings in setup.glade (XML file)
                 SETUP_GLADE="./setup.data/setup.glade"
                 ATI_LOG_EXP=$( echo ${ATI_LOG} | sed -e 's/\//\\\//g' )
