--- kcl_acpi.c	2014-12-08 15:28:55.083426066 +0800
+++ kcl_acpi_fix.c	2014-12-08 15:30:01.541280500 +0800
@@ -831,6 +831,7 @@
 
 static acpi_status KCL_ACPI_Slot_No_Hotplug(KCL_ACPI_DevHandle handle, u32 lvl, void *data, void **rv)
 {
+#if LINUX_VERSION_CODE <  KERNEL_VERSION(3,17,0)
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3,12,7)
    struct acpi_device *tdev = NULL;
    struct pci_dev *pdev = (struct pci_dev *)data;
@@ -846,6 +847,7 @@
       }
    }
 #endif
+#endif
    return 0;
 }
 
